zk.go

```diff
@@ -5,13 +5,16 @@ import (
 	"path"
 	"sort"
 	"strconv"
+	"strings"
 	"time"
 
 	"github.com/samuel/go-zookeeper/zk"
 )
 
-// ZK wraps a zookeeper connection
-type ZK struct{ *zk.Conn }
+type ZK struct {
+	conn   *zk.Conn
```

Comment:
you would save yourself from all the .conn patches if you just leave this nameless. If we want to track upstream, it's probably recommended to do that.